Client/Frontend/Browser/Browser.swift

```diff
@@ -23,6 +23,7 @@ protocol BrowserDelegate {
 
 class Browser: NSObject {
     var webView: WKWebView? = nil
+    var previousHomePanel = 0
```

Comment:
I think `lastHomePanel` would be a more appropriate name than `previousHomePanel`. "Previous" makes it sounds like we're on a current panel that's different from the one before, whereas "last" simply means the one that was last selected.